ios/Sources/Component/Tag/CutCornerRectangle.swift

```diff
@@ -0,0 +1,40 @@
+import SwiftUI
+
+struct CutCornerRectangle: Shape {
```

Comment:
Please change this `struct` 's accessor to `public` and implement `topLeft` only version?
This shape is used by Home as well. 🙏🏼  https://www.figma.com/file/D2VLqh0xOXbH0zB6cTz053/DroidKaigi_2021_official_App-(iOS)?node-id=159%3A94